wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ArchivedList/ArchivedListViewController.swift

```diff
@@ -49,40 +50,30 @@ final class ArchivedListViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
         view.accessibilityViewIsModal = true
-        view.backgroundColor = SemanticColors.View.backgroundConversationList
-        setupNavigationItem()
+        view.backgroundColor = ColorTheme.Backgrounds.surface
         setupCollectionView()
         setupEmptyPlaceholder()
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-
+        setupNavigationBarTitle(L10n.Localizable.ArchivedList.title.capitalized)
+        configureNavigationBarAppearance()
```

Comment:
I will move the first line into `init` since Apple recommends to [avoid tying the creation of bar button items in the navigation item to the creation of the view controller's view](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621851-navigationitem). I assume the same is true for the navigation item title.

The second line @agisilaos prefers to write in `viewWillAppear`.

Reply:
Oh yeah I forgot about this best practice. Please ignore my comment.